Stacktrace styling

hyperionel opened the pull request:
Solves #120

hyperionel committed:
added highlightjs for syntax highlighting and made stacktrace look better

Chocksy commented:
Ceva screenshot la cum arata? Si sa pui label cand e gata de review. 💃

hyperionel committed:
changed style and fixed some bugs

hyperionel commented:
![stacktrace](https://cloud.githubusercontent.com/assets/9285305/15213339/3e0f05b8-184f-11e6-859d-f7a1bd6a3aa4.png)

hyperionel committed:
added accordion structure to stacktrace frames

hyperionel committed:
css fix

Chocksy commented on app/assets/javascripts/epiclogger.js:
```diff
@@ -1,9 +1,14 @@
+};
+
+$(document).ready(ready);
+$(document).on('page:load', ready);
```
aici e aceeasi poveste foloseste ce ti-am dat din jquery-turbolinks si fai asa cum e acolo

Chocksy replied:
si nu baga nimic in epiclogger.js asta ar trebuii sa fie in componenta ei sau in pages.

Chocksy committed:
change how the syntax styling looks

- add the platform for the code to allow the highligher to show the right colors
- change the format of the code we show
